client/src/components/existing-request/existing-request-display.vue

```diff
@@ -276,6 +303,36 @@ export default class ExistingRequestDisplay extends Mixins(
     return ''
   }
 
+  private get reviewDate () {
+    if (this.nr.waiting_time) {
+      let waitingTime = this.nr.waiting_time
+      // make sure minimum waiting time is 1
+      if (waitingTime < 1) {
+        waitingTime = 1
+      }
+      // add 20 % for weekends to account for business days
+      waitingTime = Math.round(waitingTime * 1.2)
+      let reviewDate = new Date()
+      reviewDate.setDate(reviewDate.getDate() + waitingTime)
+      return Moment(reviewDate).format("MMM Do[,] YYYY") + " (" + this.nr.waiting_time + " days)"
+    }
+    return ""
+  }
+
+  private get queueDate () {
+    if (this.nr.oldest_draft) {
+      return Moment(this.nr.oldest_draft).format("MMM Do[,] YYYY")
```

Comment:
What is the timezone of this output? I think we decided it should always be "Pacific". Do the mockups specify to display "Pacific"?

NB: we just display Pacific instead of PDT or PST

Reply:
(Need to make sure the datetime isn't local to the user if they are in, say, UK. So we force it to Pacific.)

Reply:
Put in hard coded "Pacific Time"

Reply:
What tz does Moment convert to? Need to make sure it's Pacific Daylight/Savings Time.

Reply:
I think it's just a formatter? It's used on all of the other dates in the view :)

Reply:
My concern is that `this.nr.oldest_draft` is in UTC and Moment is not correctly converting it to Pacific. Can you confirm please? Maybe we need to use [Moment Timezone](https://momentjs.com/timezone/) to do the conversion.

Reply:
If you a test timedate, try "2021-02-25T02:04:07.934Z".
This should display as "Feb 24 6:04 PM Pacific".

Reply:
I have added moment timezone

Reply:
Does it test OK now?